package com.example.payments;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;

@EnableScheduling
@SpringBootApplication
public class PaymentSagaApplication {
    public static void main(String[] args) {
        SpringApplication.run(PaymentSagaApplication.class, args);
    }
}
